docs/usage/config-presets.md

```diff
@@ -232,3 +193,38 @@ For example the result may be:
   "extends": ["local>myorgname/.github:renovate-config"]
 }
 ```
+
+## npm-hosted presets
+
+If you manage multiple repositories using Renovate and want the same custom config across all or most of them, then you might want to consider publishing your own preset config so that you can "extend" it in every applicable repository.
```

Comment:
```suggestion
Using npm-hosted presets is deprecated, we recommend you do not follow these instructions and instead use INSERT PREFERRED METHOD HERE.

If you manage multiple repositories using Renovate and want the same custom config across all or most of them, then you might want to consider publishing your own preset config so that you can "extend" it in every applicable repository.
```

Do we want to put a warning here? I don't know what the best alternative is, but just putting this out there for discussion.

Reply:
Sounds good. I think preferred method is to use `local` presets. We should implement them for all platforms.